spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -141,5 +174,90 @@ module.exports = function (provider, setup) {
     });
   });
 
-  // TODO: add tests for tcpsocket.secure, accepting multiple.
+  it("Secures a socket", function (done) {
+    socket.connect('www.google.com', 443, function () {
+      var prepared;
+      // prepareSecure is Chrome specific, see freedom-for-chrome
+      socket.prepareSecure(function () {
+        prepared = true;
+      });
+      // TODO: timeout approach sucks, preferred approach:
```

Comment:
Makes sense - for now I'll forget about prepareSecure then. Looks like the Chrome team is nearing fixing this on their end too, there's recent activity on the issue (https://code.google.com/p/chromium/issues/detail?id=467677).